section4/exercises/classMethods.js

```diff
@@ -28,13 +28,34 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings
+  }
+
+  changeProtein(protein) {
+    this.protein = protein
+  }
+
+  addTopping(additionalToppings) {
```

Comment:
Is `additionalToppings` an array of toppings, or a single topping?  If it's a single topping, I would consider a more appropriate variable name. If it's an array of toppings, you would end up pushing an array into another array and generally you don't want to mix data types inside of arrays.